### DISCUSSION TITLE
No files on card.

### DISCUSSION THREAD

**Discussion by jarda99** [frntc/Sidekick64#73] [General]
Hi, I need some advice on Sidekick64 on C16.
I prepared an SD card, switched the jumper on the Sidekic64 board to D3 booted, but I have a problem, when I put the filebrowser on, I can't see any files on the card, I can't even browse the directories on the card, while I have files uploaded to the specified directories on the card.
Some of the pre-set programs that are part of the card work, but I can't see my files on the card, could someone tell me what I'm doing wrong?
I also tried overclocking the Rpi as mentioned on the site but nothing.....
I also edited the config.txt file according to the site uncommented sidekick264.txt and commented out #sidekick64.txt
I'm testing on an RPi3 model B V1.2

Thank you for the advice.
J.

**Comment by jarda99** [comment]
I also tested Rpi3B+ ( overclocked ), new install of the card, it still behaves the same, I can't see any files on the card, and I can't even browse the card.
Card settings:
arm_freq=1550
core_freq=650
sdram_freq=600
over_voltage=2
So, specifically, the FileBrowser which is under the letter A in the main menu does not work, I accidentally pressed F7 and got to the disk files menu.
At the same time, according to the menu, F7 is used to jump to BASIC :-)
Is there a reason why the filebrowser doesn't work?

**Reply by frntc** [reply]
To which directories did you copy your own files?

**Reply by jarda99** [reply]
On the SD card to the PRG264 directories - files with the .prg extension to the D264 directory files with the .d64 extension

EDIT:
I've tried uploading one xxx.prg file to each directory on the card, but it's still not possible to browse the directories on the card, when I start filemanager from the main menu under the A key, I'm stuck on one line and can't browse the card, it just flashes LOADING and nothing happens not happening

**Reply by frntc** [reply]
ah wait! you're launching a file manager PRG? Then you can't access files on the SD-card. I though you were referring to the build-in Sidekick264-browser!

(over)clocking is not related to any problem here.

**Reply by jarda99** [reply]
And what is the PRG file manager there for?
When I can't browse the files on the card?
Thank you.

**Reply by frntc** [reply]
You can browse files on the SD card with the SK264 internal browser. The other one is meant to using SD2IEC, Pi1541, floppy drives, ...

**Reply by jarda99** [reply]
Ok, thanks for the clarification.
